requirements.txt

```diff
@@ -1,2 +1,2 @@
 dice<=1.1.0
-pillow
\ No newline at end of file
+pillow==4.1.1
```

Comment:
Probably. Though we should also be able to get away with versions lower than that.